# Brickman Group <img src="https://github.com/brickmanlab/.github/blob/master/profile/reNEW_logo.png" width="200">

![BrickmanBanner](https://github.com/brickmanlab/.github/blob/master/profile/BrickmanBanner.png)

## Transcriptional basis for cell fate choice

The [Brickman group](https://renew.ku.dk/research/reseach-groups/brickman-group/) aims to understand the transcriptional basis for early embryonic lineage specification.

**We are interested in the dynamic mechanisms by which cells can both reversible prime towards a particular fate or undergo a transition into commitment.**

## Publications

<details>
  <summary> Selected publications </summary>

Rothová, M. M., Nielsen, A. V., Proks, M., Wong, Y. F., Riveiro, A. R., Linneberg-Agerholm, M., David, E., Amit, I., Trusina, A., and Brickman, J. M. **(2022)**. [Identification of the central intermediate in the extra-embryonic to embryonic endoderm transition through single-cell transcriptomics](https://www.nature.com/articles/s41556-022-00923-x). _Nature Cell Biology_, doi: 10.1038/s41556-022-00923-x.

Hamilton, W.B., Mosesson, Y., Monteiro, R.S., Emdal, K.B., Knudsen, T.E., Francavilla, C., Barkai, N., Olsen, J.V. and Brickman, J.M. **(2019)**. [Dynamic lineage priming is driven via direct enhancer regulation by ERK](https://www.nature.com/articles/s41586-019-1732-z). _Nature_, doi: 10.1038/s41586-019-1732-z.

Weinert, B.T., Narita, T., Satpathy, S., Srinivasan, B., Hansen, B.K., Scholz, C., Hamilton, W.B., Zucconi, B.E., Wang, W.W., Liu, W.R., Brickman, J.M., Kesicki, E.A., Lai, A., Bromberg, K.D., Cole, P.A., and Choudhary, C. **(2018)**. [Time-Resolved Analysis Reveals Rapid Dynamics and Broad Scope of the CBP/p300 Acetylome](https://www.sciencedirect.com/science/article/pii/S0092867418305269?via%3Dihub). _Cell_ 174, 231-244.e212, doi:10.1016/j.cell.2018.04.033.

Anderson, K.G.V., Hamilton, W.B., Roske, F.V., Azad, A., Knudsen, T.E., Canham, M.A., Forrester, L.M., and Brickman, J.M. **(2017)**. [Insulin fine-tunes self-renewal pathways governing naive pluripotency and extra-embryonic endoderm](https://www.nature.com/articles/ncb3617). _Nature Cell Biology_ 19, 1164-1177, doi:10.1038/ncb3617.

Nissen, S.B., Perera, M., Gonzalez, J.M., Morgani, S.M., Jensen, M.H., Sneppen, K., Brickman, J.M.*, and Trusina, A.* **(2017)**. [Four simple rules that are sufficient to generate the mammalian blastocyst](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2000737). _PLoS Biol_ 15, e2000737, doi:10.1371/journal.pbio.2000737.  *joint senior author

Migueles, R.P., Shaw, L., Rodrigues, N.P., May, G., Henseleit, K., Anderson, K.G., Goker, H., Jones, C.M., de Bruijn, M.F., Brickman, J.M., and Enver, T. **(2017)**. [Transcriptional regulation of Hhex in hematopoiesis and hematopoietic stem cell ontogeny](https://www.sciencedirect.com/science/article/pii/S0012160616306388?via%3Dihub). _Developmental Biology_ 424, 236-245, doi:10.1016/j.ydbio.2016.12.021.

Illingworth, R.S., Hölzenspies, J.J., Roske, F.V., Bickmore, W.A., and Brickman, J.M. **(2016)**. [Polycomb enables primitive endoderm lineage priming in embryonic stem cells](https://elifesciences.org/articles/14926). _Elife_ 5, doi:10.7554/eLife.14926.

Martin Gonzalez, J., Morgani, S.M., Bone, R.A., Bonderup, K., Abelchian, S., Brakebusch, C., and Brickman, J.M. **(2016)**. [Embryonic Stem Cell Culture Conditions Support Distinct States Associated with Different Developmental Stages and Potency](https://www.sciencedirect.com/science/article/pii/S2213671116301333?via%3Dihub). _Stem Cell Reports_ 7, 177-191, doi:10.1016/j.stemcr.2016.07.009.
</details>

## Datasets

[Rothova et al., (2022). Nature Cell Biology.](https://zenodo.org/record/6566016#.ZFoIu9JBxhF) Single-cell RNA-seq datasets from FOXA2<sup>Venus</sup> reporter mouse embryos and embryonic stem cell differentiation towards endoderm.
