![](https://github.com/brickmanlab/.github/blob/master/profile/BrickmanBanner.png) 

# reNEW Brickman Group
### Transcriptional basis for cell fate choice

<br>

<p align="center">
<img src="https://github.com/brickmanlab/.github/blob/master/profile/Brickman1.jpg" width="500">
</p>

<br>

The Brickman group aims to understand the transcriptional basis for early embryonic lineage specification. 

**In particular, the group is interested in dynamic mechanisms by which cells can both reversible prime towards a particular fate or undergo a transition into commitment.**

<br>

## Group research
<details>
As early mammalian development is highly regulative the derivation of embryonic stem (ES) cell lines from early embryos produces heterogeneous culture systems that can recapitulate the progenitor cell types normally present in the embryo. As a result we are able to exploit ES cells as a model to study the process of lineage choice in embryonic development and, infer from embryonic development the identity of factors that block differentiation to support ES cell self-renewal. We seek to understand the transcriptional basis of ES cell potency/pluripotency and how this relates to normal embryonic development.

#### Our primary focus has been on the specification and patterning of endoderm, both dynamically in ES cell culture, and during differentiation.

Understanding the process of endoderm specification is essential for the directed differentiation of embryonic stem (ES) cells towards specific functional cell types from liver, lung, thyroid, thymus, and pancreas. Indeed our work in this area has already generated technologies for the directed differentiation of both mouse and human ES cells. The induction of endoderm from undifferentiated precursor populations also requires the removal of key blocks to differentiation and our work also suggests that there are important links between certain negative regulators of endoderm specification and ES cell self-renewal.

As the pathways that regulate early embryonic differentiation and the factors that regulate them are particularly well conserved, we have been able to use a unique experimental combination of ES cells and Xenopus embryos to probe the nature of the regulatory networks that guide lineage specification. We have also developed a number of unique tools for the real time imaging of cell fate decisions and transcriptional plasticity in early embryos and ES cells.
</details>
<br>

## Projects
<details>
  <summary> Current and previous research projects </summary>
<br>


#### Extrinsic Signals that Regulate Lineage Priming in ES cells
We have found that ES cells are composed of two populations of self-renewing and dynamically interconverting populations of early epiblast and primitive endoderm progenitors. We are interested in the signals that drive cells from one state to another and, signals that block cells, from a primed endoderm state, from entering differentiation. We are involved with a number of collaborative projects directed at collecting quantitative real time data to understand these processes.

#### Intrinsic Factors Regulating Heterogeneity and Self Renewal in ES cells
We have observed that the expression of a large number of early endoderm and epiblast genes change as ES cells transit between an early epiblast and endoderm state. We are interested in the transcriptional mechanisms that regulate this reversible lineage priming.

#### Function of the Conserved Transcription Factor Network Downstream of Oct4
We have identified a set of targets regulated by Oct4 and its homologues in Xenopus. We are interested in the means by which these factors regulate differentiation.
 
#### Transcriptional Basis for Lineage Specification in Endoderm
We have been using a number of genomewide technologies to characterise the changes transcription factor and RNA polymerase binding associated with specific precursor populations during the progressive specification of endoderm from pluripotent ES cells. We are particularly interested in the means by which signalling pathways impact on the progressive specification of transcription factor networks.
  
#### Polarity, Self Renewal and Differentiation
We have found that one of the major effectors of Fgf signalling during ES cell differentiation towards endoderm is remodelling of the extra-cellular matrix. This matrix is able to pattern naïve endoderm at the same time as inducing polarised endodermal epithelia. We are interested in the means by adhesive information is input into transcriptional programs. We have also associated the regulation of cellular adhesion with the Oct4 target network and are interested in how changes in cellular adhesion impact on cellular potency and commitment.

<br >
  
### Previous Achievements

#### Revised Definitions of Pluripotency based on Low Level Dynamic Changes in Transcriptional States Associated with Cell Fate Choice in ES cells. 
We have developed a series of highly sensitive fluorescent reporter ES cell lines that have enabled us to define functionally distinct populations of self-renewing ES cells (Canham link, recent Current Opinions paper). These reporters employ a reiterated IRES sequence that functions as a translational amplifier to drive the expression of highly sensitive fluorescent proteins cells (see both Tsakiridis et al, NAR 2009 and Canham et al, PLoS Biol 2010) to give a highly sensitive and dynamic read out of lineage primed states in single cells.

#### A Model for Endoderm Induction and Expansion
We have established a number of additional fluorescent reporter ES cell lines alongside defined in vitro systems for embryonic differentiation. We have been able to derive, purify and expand defined anterior endoderm using these reporters. We were able to successfully expand ES cell derived endoderm for multiple passages and use our in vitro model system to uncover a new role for Fgf signalling during endoderm specification. (Morrison et al, Cell Stem Cell 2008, Livigni et al Curr Protocols In Stem Cell Biol, 2009).

#### A Defined and Conserved Oct4 Target Network
We have shown that the activity of Oct4, an essential ES cell transcription factor, is conserved in evolution and related to the activation of a conserved program of gene expression that suppresses differentiation and commitment in early development (Morrison and Brickman, Dev 2006, Hammachi et al, Cell Reporter 2012).


</details>

<br>

## Publications
<details>
  <summary> Selected publications </summary>
<br>

Rothová, M. M., Nielsen, A. V., Proks, M., Wong, Y. F., Riveiro, A. R., Linneberg-Agerholm, M., David, E., Amit, I., Trusina, A., and Brickman, J. M. **(2022)**. [Identification of the central intermediate in the extra-embryonic to embryonic endoderm transition through single-cell transcriptomics](https://www.nature.com/articles/s41556-022-00923-x). _Nature Cell Biology_, doi: 10.1038/s41556-022-00923-x.

Hamilton, W.B., Mosesson, Y., Monteiro, R.S., Emdal, K.B., Knudsen, T.E., Francavilla, C., Barkai, N., Olsen, J.V. and Brickman, J.M. **(2019)**. [Dynamic lineage priming is driven via direct enhancer regulation by ERK](https://www.nature.com/articles/s41586-019-1732-z). _Nature_, doi: 10.1038/s41586-019-1732-z.

Weinert, B.T., Narita, T., Satpathy, S., Srinivasan, B., Hansen, B.K., Scholz, C., Hamilton, W.B., Zucconi, B.E., Wang, W.W., Liu, W.R., Brickman, J.M., Kesicki, E.A., Lai, A., Bromberg, K.D., Cole, P.A., and Choudhary, C. **(2018)**. [Time-Resolved Analysis Reveals Rapid Dynamics and Broad Scope of the CBP/p300 Acetylome](https://www.sciencedirect.com/science/article/pii/S0092867418305269?via%3Dihub). _Cell_ 174, 231-244.e212, doi:10.1016/j.cell.2018.04.033.

Anderson, K.G.V., Hamilton, W.B., Roske, F.V., Azad, A., Knudsen, T.E., Canham, M.A., Forrester, L.M., and Brickman, J.M. **(2017)**. [Insulin fine-tunes self-renewal pathways governing naive pluripotency and extra-embryonic endoderm](https://www.nature.com/articles/ncb3617). _Nature Cell Biology_ 19, 1164-1177, doi:10.1038/ncb3617.

Nissen, S.B., Perera, M., Gonzalez, J.M., Morgani, S.M., Jensen, M.H., Sneppen, K., Brickman, J.M.*, and Trusina, A.* **(2017)**. [Four simple rules that are sufficient to generate the mammalian blastocyst](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2000737). _PLoS Biol_ 15, e2000737, doi:10.1371/journal.pbio.2000737.  *joint senior author

Migueles, R.P., Shaw, L., Rodrigues, N.P., May, G., Henseleit, K., Anderson, K.G., Goker, H., Jones, C.M., de Bruijn, M.F., Brickman, J.M., and Enver, T. **(2017)**. [Transcriptional regulation of Hhex in hematopoiesis and hematopoietic stem cell ontogeny](https://www.sciencedirect.com/science/article/pii/S0012160616306388?via%3Dihub). _Developmental Biology_ 424, 236-245, doi:10.1016/j.ydbio.2016.12.021.

Illingworth, R.S., Hölzenspies, J.J., Roske, F.V., Bickmore, W.A., and Brickman, J.M. **(2016)**. [Polycomb enables primitive endoderm lineage priming in embryonic stem cells](https://elifesciences.org/articles/14926). _Elife_ 5, doi:10.7554/eLife.14926.

Martin Gonzalez, J., Morgani, S.M., Bone, R.A., Bonderup, K., Abelchian, S., Brakebusch, C., and Brickman, J.M. **(2016)**. [Embryonic Stem Cell Culture Conditions Support Distinct States Associated with Different Developmental Stages and Potency](https://www.sciencedirect.com/science/article/pii/S2213671116301333?via%3Dihub). _Stem Cell Reports_ 7, 177-191, doi:10.1016/j.stemcr.2016.07.009.
</details>

<br>

## Datasets
[Rothova et al., (2022). Nature Cell Biology.](https://zenodo.org/record/6566016#.ZFoIu9JBxhF) Single-cell RNA-seq datasets from FOXA2<sup>Venus</sup> reporter mouse embryos and embryonic stem cell differentiation towards endoderm.

